VERSION

```diff
@@ -1 +1 @@
-3.1.0-b1
+3.1.0b1
```

Comment:
Is there any significance in this formatting that I don't know yet, or just an arbitrary choice?

Reply:
3.1.0-b1 is not compatible with the pip versioning and automatically converts that into 3.1.0b1. This is simply to adopt pip's versioning.